src/styles/mixins/_typography.scss

```diff
@@ -0,0 +1,9 @@
+/* Common typography-related mixins */
+
+@mixin bodyText {
+  color: var(--kui-color-text, $kui-color-text);
+  font-family: var(--kui-font-family-text, $kui-font-family-text);
+  font-size: var(--kui-font-size-30, $kui-font-size-30);
```

Comment:
I'm not sure it makes sense to include font size here; in a lot of places we actually default to 16px

Reply:
Yeah but we use this mixin only where we need too. Maybe the name isn't quite good?

Reply:
What if in some other component (say, KPrompt) we need the same styles? I think it should stay a typography mixin, it's just that the `bodyText` is too generic..